pkg/scheduler/api/node_info.go

```diff
@@ -395,13 +363,13 @@ func (ni *NodeInfo) setNode(node *v1.Node) {
 			ni.Idle.sub(ti.Resreq) // sub without assertion
 			ni.Releasing.Add(ti.Resreq)
 			ni.Used.Add(ti.Resreq)
-			ni.AddGPUResource(ti.Pod)
+			ni.SharedDevices[GPUSharingDevice].AddResource(ti.Pod)
```

Comment:
suggest to retain `AddResource` and put the `SharedDevices[GPUSharingDevice].AddResource(ti.Pod)` inside `AddResource`.